### PR TITLE
Gets the guest's name from Welcome screen

### PR DESCRIPTION
**Pull request by jennyckaplan**
This PR uses `react-navigation` params passing to get the guest's name from the NameInputView component to the Employees screen so that the guest's name is in the slack message to the host.

### CODE REVIEW

File: src/components/name-input.js [hunk 1]
```diff
@@ -10,18 +10,30 @@ import {
...
-    <TextInputWrapper>
-      <NameInputText placeholder="What&apos;s your name?" />
-    </TextInputWrapper>
-    <NextButton onPress={nextPage}>
-      <ButtonText>Next</ButtonText>
-      <WhiteArrowIcon source={ArrowIcon} />
-    </NextButton>
-  </NameInputContainer>
-);
+export default class NameInputView extends React.Component {
```
Comment (cbarber) [inline src/components/name-input.js]: Prefer a refactor commit that converts a stateless component function into a component class and then make your changes after (add constructor, state, and new function).

File: src/containers/Employees/index.js [hunk 2]
```diff
@@ -38,6 +38,9 @@ export default class Employees extends React.Component {
 
   sendNotification = slackID => {
     console.log(slackID);
+    const { navigation } = this.props;
+    const guest = navigation.getParam('guestName');
```
Comment (cbarber) [inline src/containers/Employees/index.js]: `getParam` should be defined in the propTypes.

File: src/components/name-input.js (hunk 1 above)
Comment (jennyckaplan) [reply src/components/name-input.js]: done @cbarber

Comment (cbarber) [reply src/components/name-input.js]: Almost. You should keep the original export structure. Looks like the import is using the named export and not the default.

Feature commit looks great now

**Pull request merged**